packages/ai-jsx/src/core/render.ts

```diff
@@ -311,19 +311,10 @@ async function* renderStream(
     const logImpl = renderingContext.getContext(LoggerContext);
     const renderId = uuidv4();
     try {
-      /**
-       * This approach is pretty noisy because there are many internal components about which the users don't care.
-       * For instance, if the user writes <ChatCompletion>, that'll generate a bunch of internal helpers to
-       * locate + call the model provider, etc.
-       *
-       * To get around this, maybe we want components to be able to choose the loglevel used for their rendering.
-       */
-      logImpl.log('debug', renderable, renderId, 'Start rendering element');
       const finalResult = yield* renderingContext.render(
         renderable.render(renderingContext, new BoundLogger(logImpl, renderId, renderable), appendOnly),
         recursiveRenderOpts
       );
-      logImpl.log('debug', renderable, renderId, { finalResult }, 'Finished rendering element');
```

Comment:
If we take this out, is there any other way to get this signal that doesn't involve OpenTel?

Reply:
I would be fine removing this if we replaced it with some sort of an "OpenTel console logger". That would be nicer, because we would be unifying the logging. And it would preserve visibility for people who can't/don't want to run Docker and have an external viewer like Jaeger or DD set up.

Reply:
There's a straightforward-ish way to set up a collector so that it emits things like:
```
Span #2
    Trace ID       : 48b03803a7fd4c6beea750037e9a846f
    Parent ID      : 939c625fdad147a9
    ID             : fd2a38dae778b6c9
    Name           : <OpenAICompletionModel>
    Kind           : Internal
    Start time     : 2023-08-18 16:09:22.015000064 +0000 UTC
    End time       : 2023-08-18 16:09:25.32702592 +0000 UTC
    Status code    : Ok
    Status message : 
Attributes:
     -> ai.jsx.tag: Str(OpenAICompletionModel)
     -> ai.jsx.tree: Str(<OpenAICompletionModel @memoizedId=1 model="text-davinci-003" maxTokens={100}>
  {"Q: Why is the sky blue?"}
  {"\n"}
  {"A:"}
</OpenAICompletionModel>)
     -> ai.jsx.result: Str( The sky is blue because molecules in the Earth's atmosphere scatter sunlight in all directions. Blue light is scattered more than other colors because it travels as shorter, smaller waves. This is why we see a blue sky most of the time.)
```
to the console.

Another alternative is to configure the OTel SDK to write spans to the console, though obviously that doesn't work well if the console is also being used for the inspector.

I guess my high level take is that I think these logs are so noisy that sifting through them _without_ a UI is of limited utility (or at least significantly _less_ useful) but that's certainly a judgment call.

Mainly I wanted to see if you were (at all) reluctant to see this go, which it sounds like you are, so I can bring them back. :)

Reply:
Yes, in my judgment, the logs are noisy, but sifting through them is actually not that hard, and without them, setting up OpenTel is a meaningful burden. 

`ai-jsx-program | grep MyComponentName | npx pino-pretty` drops a lot of the noise for me.